### PR TITLE
removed internal API usage from ExplodeTile and ViewSetup

### PR DESCRIPTION
**Pull request by dustinlebsock**

### CODE REVIEW

File: src/frontend-samples/explode-sample/ExplodeTile.ts [hunk 1]
```diff
@@ -586,19 +586,12 @@ export class ExplodedGraphicsTile extends Tile {
...
-        //
-      }
-    }
+
+    // When for next iModel.js update, the API will include an argument for batching options.  This should be set to the contentId.
+    const graphic = await readElementGraphics(data, tree.iModel, tree.modelId, tree.is3d);
 
     (this.tree as ExplodeTileTree).elementContentLoaded(this.data.elementId);
-    return content;
+    return { isLeaf: true, graphic } as TileContent;
```
Comment (pmconne) [inline src/frontend-samples/explode-sample/ExplodeTile.ts:594]: Remove `as TileContent`; it bypasses the type-checker.

File: src/api/viewSetup.ts [hunk 2]
```diff
@@ -138,9 +134,11 @@ export class ViewSetup {
     if (!iModel.isClosed) {
       const selectRelevantCategories = `SELECT ECInstanceId FROM BisCore.SpatialCategory ${categoryCodes.length > 0 ? `WHERE CodeValue IN ('${categoryCodes.join("','")}')` : ""}`;
       for await (const row of iModel.query(selectRelevantCategories)) {
```
Comment (pmconne) [inline src/api/viewSetup.ts:136]: Would serve as a better (more efficient) example if you did this with a single ECSql statement.

**Review by pmconne — CHANGES_REQUESTED** [review]

File: src/api/viewSetup.ts (hunk 2 above)
Comment (dustinlebsock) [reply src/api/viewSetup.ts]: Combined into single statement.

File: src/frontend-samples/explode-sample/ExplodeTile.ts (hunk 1 above)
Comment (dustinlebsock) [reply src/frontend-samples/explode-sample/ExplodeTile.ts]: TileContent cast removed.

**Review by pmconne — APPROVED** [review]

**Review by Ellord207 — APPROVED** [review]

**Review by BaoTon — APPROVED** [review]
LGTM

**Pull request merged**